mailpoet/lib/Migrations/Db/Migration_20230831_143755_Db.php

```diff
@@ -40,8 +40,9 @@ public function run(): void {
     $this->connection->executeStatement("UPDATE $logsTable SET status = 'complete' WHERE status = 'completed'");
 
     // fix empty values for errors and data
+    $this->connection->executeStatement("ALTER TABLE $logsTable CHANGE `data` `data` longtext NOT NULL AFTER run_number");
     $this->connection->executeStatement("UPDATE $logsTable SET data = '{}' WHERE data = '[]' OR data IS NULL");
-    $this->connection->executeStatement("UPDATE $logsTable SET error = '{}' WHERE error = '[]' OR error IS NULL");
+    $this->connection->executeStatement("UPDATE $logsTable SET error = NULL WHERE error = '[]' OR error IS NULL");
```

Comment:
nitpick but
`SET error = NULL WHERE error = '[]' OR error IS NULL`

I think`OR error IS NULL` is not needed